command_handler.py

```diff
@@ -46,7 +46,7 @@ def __init__(self):
       'deletestrikes': ( strikeHandler, None),
       'getstrikes': ( strikeHandler, "prints the current strike count"),
       'hotdognothotdog': ( hotdognothotdog.HotdogNotHotdog(), "It's a hotdog ... or not a hotdog"),
-      'dice': (dice.Dice()), "rolls dice, use XdY format"),
+      'roll': (dice.Roll()), "rolls dice, use XdY format"),
```

Comment:
Commands should be in the format 'commandname.CommandHame()', can you please change

Reply:
Done